File: findGCD/findgcd.py

def findgcd(a:int, b:int)-> int:
    while(b!=0):
        remainder=a%b
        a=b
        b=remainder
    return a 

def main()->None:
   result= findgcd(48,18)
   print(f"GCD is {result}")

if __name__=="__main__":
    main()

File: findGCD/findgcd.java
public class findgcd{
    //
    public static int gcd(int a, int b){
        while(b!=0){
            int remainder=a%b;
            a=b;
            b=remainder;
        }
        return a;
    }

    public static void main(String[] args){
        int result=gcd(48,18);
        System.out.println(("GCD is "+result));
    }
}